aperturedb/ImageDataCSV.py

```diff
@@ -19,126 +21,33 @@
 IMG_FORMAT    = "format"
 
 
-class ImageDataCSV(CSVParser.CSVParser):
-    """**ApertureDB Image Data.**
-
-    This class loads the Image Data which is present in a csv file,
-    and converts it into a series of aperturedb queries.
-
-
```

Comment:
what is being done here is no change to the ImageDataCSV, but the core image actions are being split into "ImageDataProcessor" which allows other classes to use them.

Reply:
How about this ? 
ImageDataCSV could be like a factory, and Create, update, delete could be arguments to create the parser ?